op_builder/fp_quantizer.py

```diff
@@ -49,23 +49,25 @@ def is_compatible(self, verbose=False):
             import triton
         except ImportError:
             if verbose:
-                self.warning(f"please install triton==2.3.0 or 2.3.1 if you want to use the FP Quantizer Kernels")
+                self.warning(f"please install triton==2.3.0,  2.3.1 or 3.0.0 if you want to use the FP Quantizer Kernels")
             return False
 
-        # triton 2.3.0 and 2.3.1 are okay and the only versions released in 2.3.x before 3.x was released
+        # triton 2.3.{0,1} and 3.0.0 are ok.
+        allowed_versions = ("2.3", "3.0")
         if pkg_version:
-            allowed = pkg_version.parse("2.3")
+            allowed = (pkg_version.parse(v) for v in allowed_versions)
             installed_triton = pkg_version.parse(triton.__version__)
-            triton_mismatch = installed_triton.major != allowed.major or installed_triton.minor != allowed.minor
+            triton_mismatch = all(installed_triton.major != a.major or installed_triton.minor != a.minor for a in allowed)
```

Comment:
Hi @siddartha-RE - could you run the pre-commit formatter to pass the formatting checks?

```
pre-commit run --all-files
```

It looks like it is failing on this line and perhaps another

Reply:
Hi @siddartha-RE - looks like there is still a formatting check failure

Reply:
Sorry about this, for some reason I am having trouble with pre-commit script in my docker container. I will sort it out and try before pushing a fix. I was hoping that the issue in the last run was the only problem.

Reply:
No problem, I tried to push to your branch but don't have permissions.  It appears to be a line length issue on lines 52 and 60

Reply:
export SETUPTOOLS_USE_DISTUTILS=stdlib
was needed for pre-commit to run in our container.

Reply:
Thanks, look like it is working now.